micall/tests/test_sample.py

```diff
@@ -154,3 +156,19 @@ def test_pickle(self):
         data = pickle.dumps(sample)
 
         self.assertNotEqual(b'', data)
+
+
+@pytest.mark.parametrize(
```

Comment:
Nice, I learnt something from this!

Reply:
Yes, it's really efficient for testing simple input/output functions.